docs/developer-guide/using-modules.md

```diff
@@ -75,7 +75,7 @@ This is a close match for the recommended minimum support in the [FT browser sup
 
 ### Customising your cuts the mustard test
 
-Origami components declare their minimum requirements in terms of [Modernizr](http://modernizr.com/docs/) tests.  Component developers are required to ensure that any JavaScript bundled with their module will run without error in all the browsers that pass the recommended CTM test above, but they may enhance their component's behaviour using more cutting edge features.  To verify the exact set of browser features required by the set of modules you are using:
+Origami components declare their minimum requirements in terms of [Polyfill Service](http://cdn.polyfill.io/v1/docs/) features. In case it doesn't exist, like with CSS features, they can be declared as [Modernizr](http://modernizr.com/docs/) tests. Component developers are required to ensure that any JavaScript bundled with their module will run without error in all the browsers that pass the recommended CTM test above, but they may enhance their component's behaviour using more cutting edge features.  To verify the exact set of browser features required by the set of modules you are using:
```

Comment:
"In case it doesn't exist" => "If the feature required is not in the polyfill service library, which is most often the case for CSS features"